Project.toml

```diff
@@ -38,9 +39,10 @@ BifurcationKit = "0.3"
 DataStructures = "0.18"
 DiffEqBase = "6.83.0"
 DocStringExtensions = "0.8, 0.9"
+DynamicQuantities = "0.13.2"
 Graphs = "1.4"
-JumpProcesses = "9.3.2"
```

Comment:
Not sure why this is marked as a change. This is how the Project.toml file looks like here:
```

DocStringExtensions = "0.8, 0.9"
DynamicQuantities = "0.13.2"
Graphs = "1.4"
HomotopyContinuation = "2.9"
JumpProcesses = "9.3.2"
LaTeXStrings = "1.3.0"
Latexify = "0.14, 0.15, 0.16"
```